Lombiq.Tests.UI/Extensions/ElementRetrievalUITestContextExtensions.cs

```diff
@@ -101,6 +104,38 @@ public static void SucccessMessageExists(this UITestContext context, string matc
                 .Text
                 .ShouldBe(errorMessage);
 
+        /// <summary>
+        /// Retrieves the elements according to <paramref name="by"/> and matches their text content against <paramref
+        /// name="toMatch"/>. The strings are trimmed in both collections. If an item in <paramref name="toMatch"/> is
```

Comment:
*both* collections?

Reply:
The results and toMatch are both trimmed. This is because if you accidentally left a space in the toMatch it would be hell to debug.

Reply:
I see. It was unclear to me what the other collection is. So maybe:

```c#
        /// Retrieves the elements according to <paramref name="by"/> and matches their text content against <paramref
        /// name="toMatch"/>. Both the text contents and <paramref name="toMatch"/> strings are trimmed.. If an item in
        /// <paramref name="toMatch"/> is <see langword="null" /> it's ignored among the result elements too. Every
        /// other item is converted to string, using invariant culture where possible.
```